src/Microsoft.ML/Runtime/Internal/Tools/CSharpApiGenerator.cs

```diff
@@ -564,7 +564,7 @@ private static string GetSymbolFromType(Dictionary<string, string> typesSymbolTa
 
             Contracts.Assert(typesSymbolTable.Select(kvp => kvp.Value).All(str => string.Compare(str, name) != 0));
 
-            return name;
+            return "Microsoft.ML." + name;
```

Comment:
>"Microsoft.ML." + name [](start = 19, length = 22)

>Fix Generator issues. This is change is a hack and will be addressed in a different PR.
Can you add comment why you doing this?

Reply:
Can you create issue about it?

---
In reply to: [189730873](https://github.com/dotnet/machinelearning/pull/61#discussion_r189730873) [](ancestors = 189730873)

Reply:
Done https://github.com/dotnet/machinelearning/issues/202